README.md

```diff
@@ -29,3 +29,30 @@ seat.to # parent node. nil for the final node
 seat.position # node position id
 seat.payload # custom payload that can be also seeded via BracketGraph::Graph#seed
 ```
+
+## Double Elimination Winner Bracket
+
+```ruby
+BracketGraph::DoubleEliminationGraph.new(bracket_size)
+```
+
+## About the Graph object
```

Comment:
If we should have two "About the Graph object" paragraphs, these should have a minor heading number (so use ### for both paragraphs, instead of a ##)